docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -446,9 +444,95 @@ Where,
 Iron Fortification
 ~~~~~~~~~~~~~~~~~~
 
+Population Coverage Data and Coverage Algorithm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The baseline coverage for iron fortification is the same as the baseline 
+coverage for folic acid fortification, as described below_.
+
+.. note:: 
+
+	A limitation of our population coverage data is that we assume no baseline 
+	iron fortification occurs in the absence of folic acid fortification and 
+	vice versa.
+
+Additionally, the coverage algorithm should be implemented in the same way as 
+for folic acid fortification, as described here_.
```

Comment:
It might be worth pulling out the common descriptions for iron and folic acid and putting it in a separate section that precedes both, but we can worry about that later.